backend/http_settings.go

```diff
@@ -273,7 +274,7 @@ func parseHTTPSettings(jsonData json.RawMessage, secureJSONData map[string]strin
 
 		if key, exists := dat[headerNameSuffix]; exists {
 			if value, exists := secureJSONData[headerValueSuffix]; exists {
-				s.Headers[key.(string)] = value
+				s.Header[key.(string)] = append(s.Header[key.(string)], value)
```

Comment:
I'd change this to:
```go
s.Header.Add(key.(string), value)
```
This has the side effect that the case of the header keys are changed (so you need to adapt the test), but that's fine since keys in HTTP headers are case insensitive.

Reply:
@andresmgot Done. Back to you ⚽ !